neusomatic/python/call.py

```diff
@@ -418,10 +418,10 @@ def call_neusomatic(candidates_tsv, ref_file, out_dir, checkpoint, num_threads,
     # 1. filter out unnecessary keys
     # pretrained_state_dict = {
     #     k: v for k, v in pretrained_state_dict.items() if k in model_dict}
-    if "module." in pretrained_state_dict.keys()[0] and "module." not in model_dict.keys()[0]:
+    if "module." in list(pretrained_state_dict.keys())[0] and "module." not in list(model_dict.keys())[0]:
```

Comment:
Can't it just be `if "module." in pretrained_state_dict and "module." not in model_dict:`?

Reply:
@marghoob We are not checking that "module" is in keys(), but wether it is in the first key (as string).